src/ploneintranet/workspace/browser/tiles/workspaces.py

```diff
@@ -41,7 +41,7 @@ def my_workspaces(context):
         path=ws_path,
     )
     workspaces = []
-    for brain in brains:
+    for brain in brains[:5]:
         css_class = escape_id_to_class(brain.getId)
```

Comment:
If you move the [:5] slice up to line 25 the main workspace listing will not be affected.